traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/dns/SignatureManager.java

```diff
@@ -72,7 +72,15 @@ protected void destroy() {
 		}
 	}
 
+	public void refreshKeyMap() {
+		initKeyMap(true);
+	}
+
 	private void initKeyMap() {
+		initKeyMap(false);
+	}
+
+	private void initKeyMap(final boolean suppressReload) {
```

Comment:
Can we rename `suppressReload` to `doReload` and invert the conditions so they read like this (avoiding double-negative)?
```
if (doReload) {
    reload();
}
```

Reply:
done. Renamed method to 'updateKeyMap' and change flag to 'reload'.